packages/fluentui/react-northstar/src/themes/teams/components/Label/labelStyles.ts

```diff
@@ -35,9 +66,17 @@ export const labelStyles: ComponentSlotStylesPrepared<LabelStylesProps, LabelVar
       ...(hasStartElement && {
         marginLeft: pxToRem(3),
       }),
+      ...(!hasStartElement &&
+        p.circular && {
+          marginLeft: pxToRem(4),
+        }),
```

Comment:
Could you please summarize the margin changes in the changelog ? It would be beneficial in the future for adoption notes

Reply:
WDYM by summarizing?

Reply:
The PR description currently only mentions fixing color schemes. Unless you looked at the code in PR, you would not expect spacing changes

Reply:
Perhaps something like

`Added left padding for circular label`

Reply:
Dang. I did that but it was lost somewhere during rebasing I think. Will fix.

Reply:
To be clear, I'm talking about the PR description, not the changelog. We can accept simple change statements in the log. But when reading the PR description in github we should see more details on the change without having to scan the code as much as possible

Reply:
Thank you. Could you be more specific next time when talking about changelog though.